assets/prefabs/Equipments/slimepod.prefab

```diff
@@ -1,13 +1,16 @@
 {
     "parent": "engine:iconItem",
     "DisplayName": {
-        "name": "Slime Pod Launcher"
+        "name": "slimepod"
```

Comment:
Again same as the other comment about the names

EDIT: See comment below about upgrade command